doc/source/servers/sklearn.md

```diff
@@ -5,7 +5,10 @@ If you have a trained SKLearn model saved as a pickle you can deploy it simply u
 Pre-requisites:
 
   * The model pickle must be saved using joblib and presently be named `model.joblib`
-  * We presently use sklearn version 0.23.2. Your pickled model must be compatible with this version
+  * Installed dependencies (may not work if versions don't match):
+      + sklearn == 0.23.2
```

Comment:
nit: maybe use the full package name (i.e. `scikit-learn`) here?

Reply:
Good catch!